lib/auth/auth.go

```diff
@@ -485,13 +487,20 @@ func (a *Server) GetClusterCACert() (*LocalCAResponse, error) {
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	cert, _, err := a.keyStore.GetTLSCertAndSigner(hostCA)
-	if err != nil {
-		return nil, trace.Wrap(err)
+
+	certs := services.GetTLSCerts(hostCA)
+	if len(certs) < 1 {
+		return nil, trace.NotFound("no tls certs found in host CA")
+	}
+
+	allCerts := certs[0]
+	for i := 1; i < len(certs); i++ {
+		allCerts = append(allCerts, '\n')
+		allCerts = append(allCerts, certs[i]...)
 	}
```

Comment:
is this a `bytes.Join(certs, []byte("\n"))`?

Reply:
Yep, thanks